doctrine/doctrine-bundle/2.4/config/packages/doctrine.yaml

```diff
@@ -1,6 +1,6 @@
 doctrine:
     dbal:
-        # IMPORTANT: DATABASE_URL *must* define the server version
+        # IMPORTANT: this "url" setting *must* define the "serverVersion" if you want to build the cache offline
```

Comment:
The alternative is to configure the `server_version` key separately if you cannot have it in the URL (which is a lot more maintainable than trying to concatenate extra query string parameters here if the env variable does not contain it).

Reply:
Yes, that's what I meant in the referenced comment, but I feel like there is a will to stop "publicizing" the separate `server_version` setting?